.github/workflows/scheduled_test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        database: [ 'mysql:5.5', 'mysql:5.6', 'mysql:5.7', 'mariadb:5.5', 'mariadb:10.0', 'mariadb:10.1', 'mariadb:10.2', 'mariadb:10.3', 'mariadb:10.4', 'mariadb:10.5', 'mariadb:10.6', 'mariadb:10.7', 'mariadb:10.8', 'mariadb:10.9' ]
+        database: [ 'mysql:5.5', 'mysql:5.6', 'mysql:5.7', 'mysql:8.0', 'mariadb:5.5', 'mariadb:10.0', 'mariadb:10.1', 'mariadb:10.2', 'mariadb:10.3', 'mariadb:10.4', 'mariadb:10.5', 'mariadb:10.6', 'mariadb:10.7', 'mariadb:10.8', 'mariadb:10.9' ]
```

Comment:
MySQL 8.0 test action took longer than  others (1.5 hours while others took 10 to 15 mins)